### ISSUE TITLE
Manual mode unusable due to buggy date and time entry

### ISSUE THREAD

**Issue by victoriapurplepixel** [toggl/track-macos-feedback#40]
## Summary
In manual mode it's sometimes impossible to enter the date and time in the date time picker. The old version of the app also had a calendar for the date picker which would be helpful in the new app.

Write an X between [ ] to select an option.
Leave this untouched if none applies.
-->
- [ ] Crash 
- [X ] Glitch 
- [ ] Sync 

## Affected versions

Where does this happen?
Version 8.9.0 (2066)
MacBook Pro 2021, Monterey 12.6

**Comment by HeRovaZy** [comment]
Yes, quite frustrating, especially if you want to start from eg 09:00, it doesn't register 00 as valid most of the time.

**Comment by AndrewVebster** [comment]
hey 👋 Just letting you know that we recently updated the UX for time/duration editing. App version is 8.10.0. I hope it improves your experience. If you still see any issues or have suggestions, feel free to leave us a comment here or by using the application feedback form 🙌
